chart/epinio/values.yaml

```diff
@@ -53,7 +53,7 @@ certManagerNamespace: cert-manager
 # Connection details for the S3 storage
 s3:
   endpoint: s3.amazonaws.com
-  bucket: ""
+  bucket: "epinio"
```

Comment:
I would not change the default value for the bucket

Reply:
I don't agree because the previous logic was still setting `epinio` as default but it was hiding that information inside a non explicit way for the user:
 
`{{- $s3Bucket := .Values.minio.enabled | ternary "epinio" .Values.s3.bucket -}}`

In the new way, the user is clearly seeing what is the default in the values.